packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.test.ts

```diff
@@ -0,0 +1,300 @@
+        expect(social_share.popoverButtonIconScale).toBe('s');
+
+    });
+    test('test displayTipText and setting defualt values', async () => {
+        expect(social_share.displayTipText).toBe(true);
+       
+        social_share.displayTipText = false;
+        expect(social_share.displayTipText).toBe(false);
+
+    });
+    test('test shortenShareUrl and setting defualt values', async () => {
+        expect(social_share.shortenShareUrl).toBe(true);
+       
+        social_share.shortenShareUrl = false;
+        expect(social_share.shortenShareUrl).toBe(false);
+
+    });
+    test('test socialMedia and setting defualt values', async () => {
+        expect(social_share.socialMedia).toBe(true);
+       
+        social_share.socialMedia = false;
+        expect(social_share.socialMedia).toBe(false);
+
+    });
+    test('test shareIconsLayout and setting defualt values', async () => {
+        expect(social_share.shareIconsLayout).toBe('vertical');
+       
+        social_share.shareIconsLayout = 'horizontal';
+        expect(social_share.shareIconsLayout).toBe('horizontal');
+
+    });
+    test('test scale and setting defualt values', async () => {
+        expect(social_share.scale).toBe('m');
+       
+        social_share.scale = 'l';
+        expect(social_share.scale).toBe('l');
+
+    });
+    test('test successMessage and setting defualt values', async () => {
+        expect(social_share.successMessage).toBe('');
+       
+        social_share.successMessage = 'Your awsome map has been copied to clipboard';
+        expect(social_share.successMessage).toBe('Your awsome map has been copied to clipboard');
+
+    });
+    //defaultUrlParams
+    test('test defaultUrlParams and setting defualt values', async () => {
+        expect(social_share.defaultUrlParams).toBe(null);
+        const shareParams =  {
+            center: true,
+            level: true,
+            viewpoint : true,
+            selectedFeature: false,
+            hiddenLayers: true
+        } 
+        social_share.defaultUrlParams = shareParams;
+        
+        expect(social_share.defaultUrlParams).toBe(shareParams);
+
+    });
+    //popoverPositioning
+    test('test popoverPositioning and setting defualt values', async () => {
+        expect(social_share.popoverPositioning).toBe('absolute');
+       
+        social_share.popoverPositioning = 'fixed';
+        expect(social_share.popoverPositioning).toBe('fixed');
+
+    });
+
+    test('test removePopoverOffset and setting defualt values', async () => {
+        expect(social_share.removePopoverOffset).toBe(false);
+       
+        social_share.removePopoverOffset = true;
+        expect(social_share.removePopoverOffset).toBe(true);
+
+    });
+});
+
+
+
+describe('test inline', async () => {
+    const social_share = document.createElement('instant-apps-social-share');
+    social_share.setAttribute('mode','inline');
+    social_share.setAttribute('id','create-inline');
```

Comment:
why are you setting an id here?

Reply:
but within the context of this test, the `id` isn't being used. doesn't seem necessary here

Reply:
I agree I will change it